### ISSUE TITLE
related host: immediately create ns record

### ISSUE THREAD

**Issue by ThomasWaldmann** [nsupdate-info/nsupdate.info#302]
Currently, when adding a related host, the ns record won't be created immediately, but only after first dyndns update of the primary host (which can take a while when primary IP changes rarely).

There is a workaround via creating the update via the Web UI (host view, you can put arbitrary IPs there), but doing it automatically would be nice.

**Comment by gschintgen** [comment]
I'm not even sure that the related hosts feature is working any longer *at all* for adding new hosts to an existing master.

I've just added a related host (on nsupdate.info) that I wanted to be IPv6-only, but after adding it the WebUI is showing the following for my new host:
```
IPv4 address: none      [expected result]
IPv4 interface ID: none [I left the field empty, so this is expected]
IPv6 Address: none      [UNEXPECTED!]
IPv6 interface ID: ::216......... [the configured ID]
```
I then tried updating ("nochg") the IPv6 of the master host. Nothing changed.
I added a few test hosts with an IPv4 Interface ID set to 0 (even though I don't want IPv4 for the actual host that I tried adding). Nothing.
I tried updating the IPv4 of the master. No change.
I unchecke the "Available" flag, updated, re-enabled, updated again. No success.

I can't get this to work.
BUT: **All the (dual-stack) hosts that I previously configured for the very same master host show up just fine** in the list of related hosts! Maybe there's something completely non-obvious that I'm missing and that I forgot since the last time I added a host, but seems doubtful to me.

**Comment by gschintgen** [comment]
Oh well. The completely non-obvious step that I missed is that I had to do a fake update to a wrong address first! And then change back to the actual address. So it's not even about giving a nudge, but rather a shove to get the service to set related hosts up properly.
I really don't remember doing that for my previous hosts. Possibly even some regression?

**Comment by ThomasWaldmann** [comment]
Yes, that is because no-change updates sent to the service don't trigger dns zone updates (neither for main host nor for any related hosts).

So, the workaround is to update to a *different* IP and then update again to the *desired* IP via the web interface.
